Formula/dnsviz.rb

```diff
@@ -14,8 +14,10 @@ class Dnsviz < Formula
     sha256 "33cb4ee29142ae28d442e37dc4df4cb0c659a1734ad83777682b909d008151ca" => :sierra
   end
 
+  depends_on "bind" => [ :optional, :test ]
```

Comment:
We can no longer accept optional dependencies. It has be the default or not included or part of the build or test.

Reply:
Okay, fixed (along with a few other things that `brew audit` caught, which I hadn't fixed with the last `git push -f`).

You might want the wording in the [cookbook documentation](https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md) to be a little strong to indicate that optional dependencies are no longer a thing (`brew audit` caught it though).